hot/dev-server.js

```diff
@@ -40,10 +40,15 @@ if (module.hot) {
 						"warning",
 						"[HMR] Cannot apply update. Need to do a full reload!"
 					);
-					log("warning", "[HMR] " + err.stack || err.message);
+					if (status === "abort") {
```

Comment:
Stack should be printed. Maybe create a helper `errorStack(err)` which gets the message + stack. It can check if stack contains the message to avoid duplication..